CMakeLists.txt

```diff
@@ -103,14 +91,8 @@ foreach(arch IN LISTS HOST_ARCH CUDA_ARCH)
   add_dependencies(${CORE_LIBRARY_NAME} "generate_llvm_runtime_${arch}")
 endforeach()
 
-
-FILE(WRITE ${CMAKE_CURRENT_LIST_DIR}/taichi/common/version.h
-        "#pragma once\n"
-        "#define TI_VERSION_MAJOR \"${TI_VERSION_MAJOR}\"\n"
-        "#define TI_VERSION_MINOR \"${TI_VERSION_MINOR}\"\n"
-        "#define TI_VERSION_PATCH \"${TI_VERSION_PATCH}\"\n"
-        "#define TI_CUDAVERSION \"${CUDA_VERSION}\"\n"
-        )
+configure_file(taichi/common/version.h.in version.h)
```

Comment:
Can we somehow still keep these headers in taichi/common folder? ;)

Reply:
Sure. Which one do you think is better?
1. add `.in` suffix to distinguish the files before and after processing
2. or just replace them in place

Reply:
@sjwsl let's keep the `.in` suffix so that we know what happened there :D Thanks a lot!

Reply:
Got it. But I accidentally deleted my local branch so can't update this PR now... Could you help merge this first and then I will give another PR for this. Thanks!

Reply:
sure!